fcrepo-http-api/src/main/java/org/fcrepo/http/api/ContentExposingResource.java

```diff
@@ -192,7 +193,7 @@ public Triple apply(final Statement input) {
         }
         servletResponse.addHeader("Vary", "Accept, Range, Accept-Encoding, Accept-Language");
 
-        return Response.ok(rdfStream).build();
+        return ok(rdfStream.withThisContext(rdfStream.transform(new Deskolemizer(idTranslator, null)))).build();
```

Comment:
Thanks for the further thought, @ajs6f. Let's go with your assumption until we have a pressing need to do otherwise.